openmc/plots.py

```diff
@@ -1199,6 +1322,8 @@ def to_xml_element(self):
             color = _SVG_COLORS[color.lower()]
         subelement.text = ' '.join(str(x) for x in color)
 
+        self._check_domains_consistent_with_color_by(self._wireframe_domains)
```

Comment:
This method already has access to `self._wireframe_domains`, so we can remove the argument from the method.

Reply:
Additionally, even though it's being accessed within the class implementation, using the property rather than the underlying attribute whenever possible is preferred.

Reply:
Oh I see. Missed that. Thanks!